docs/_build/html/.buildinfo

```diff
@@ -1,4 +1,4 @@
 # Sphinx build info version 1
 # This file hashes the configuration used when building these files. When it is not found, a full rebuild will be done.
```

Comment:
Curious as to why the build of the doc is not ignored in `.gitignore`? 

Do you want to keep an html version of the doc in the repo? 

Maybe a pdf version would be better (`make latxpdf`)? Or if it has to be html then a single page (`make singlehtml`)

Reply:
A fundamental misunderstanding of how readthedocs.io works, I thought we would be pushing an html directory tree to it. It's getting added to the list of clean ups.